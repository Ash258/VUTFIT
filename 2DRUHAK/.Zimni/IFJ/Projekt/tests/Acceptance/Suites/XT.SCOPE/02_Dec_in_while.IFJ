' Declaration in while block

scope
    dim counter as integer = 10

    do while counter < 15
        dim a as integer = 669878
        print a;
        counter = counter + 1
    loop
    print !"\n";
    counter = 100
    do while counter < 106
        dim a as integer = 1234
        print a;
        counter = counter + 1
    loop
end scope
