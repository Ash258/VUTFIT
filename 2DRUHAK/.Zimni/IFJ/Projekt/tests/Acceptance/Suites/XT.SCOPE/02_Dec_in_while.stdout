 669878 669878 669878 669878 669878
 1234 1234 1234 1234 1234 1234